Android/app/src/main/java/org/opendroneid/android/app/DebugActivity.java

```diff
@@ -169,6 +171,21 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 message = getString(R.string.Logging_not_activated);
             showToast(message);
             return true;
+        } else if (id == R.id.maptypeNORMAL) {
+            if (item.isChecked()) item.setChecked(false);
+            else item.setChecked(true);
+        } else if (id == R.id.maptypeSATELLITE) {
+            if (item.isChecked()) item.setChecked(false);
+            else item.setChecked(true);
+        } else if (id == R.id.maptypeTERRAIN) {
+            if (item.isChecked()) item.setChecked(false);
+            else item.setChecked(true);
+        } else if (id == R.id.maptypeHYBRID) {
+            if (item.isChecked()) item.setChecked(false);
+            else item.setChecked(true);
+        } else if (id == R.id.maptypeNONE) {
+            if (item.isChecked()) item.setChecked(false);
+            else item.setChecked(true);
```

Comment:
The above change is a good idea. 

The code can be simplified a bit though. Android studio gives you a suggestion on that if you hover the mouse of the "if" parts. Doing that simplification also gets rid of the strangely formatted if-else clauses, which are now hard to read because things are squeezed onto only two lines instead of four.

At app startup, it is not indicated which map type is currently in use. This can fixed by inserting the below line here:
https://github.com/opendroneid/receiver-android/blob/master/Android/app/src/main/java/org/opendroneid/android/app/DebugActivity.java#L97

`menu.findItem(R.id.maptypeHYBRID).setChecked(true); // Configured in AircraftMapView.setMapSettings()`